torch_geometric/sampler/base.py

```diff
@@ -610,7 +610,7 @@ def sample(
         return torch.multinomial(weight, num_samples, replacement=True)
 
 
-class BaseSampler(ABC):
+class BaseSampler:
```

Comment:
I think we can keep this and make `sample_from_nodes` and `sample_from_edges` abstractmethod.